### DISCUSSION TITLE
Setting the application icon

### DISCUSSION THREAD

**Discussion by axtens** [unoplatform/uno#14526] [Q&A] [answered]
```
    <PropertyGroup>
        <LangVersion>latest</LangVersion>
        <UnoUIUseRoslynSourceGenerators>true</UnoUIUseRoslynSourceGenerators>
        <ApplicationIcon>Peloton_Logo_MarkOnly_TransparentBack-lighter.ico</ApplicationIcon>
    </PropertyGroup>
```
is what I have in my .csproj. The .ico file is in the same folder as the .csproj. As nice as the Uno icon is, I'd rather have our own showing. What other things need to be done to make our icon visible in the taskbar on launch?

**Comment by jeromelaban** [comment]
Thanks for the question. Can you specify which platform head you are referring to?

**Reply by axtens** [reply]
.Windows

**Reply by pictos** [reply]
@axtens any chance you share a github link to a sample app that reproduces the issue or share the binlog.
If you reference Resizetizer there's a chance that it overrides your setting

**Reply by axtens** [reply]
@pictos Here's the binlog
[msbuild.binlog.zip](https://github.com/unoplatform/uno/files/13454764/msbuild.binlog.zip)

**Reply by axtens** [reply]
And another generated via the Visual Studio extension
[PelotonIDE.Windows_Debug_x64_DesignTimeBuild_2023-11-24T11_17_26.0289179+08_00.zip](https://github.com/unoplatform/uno/files/13454834/PelotonIDE.Windows_Debug_x64_DesignTimeBuild_2023-11-24T11_17_26.0289179%2B08_00.zip)

**Reply by pictos** [reply, marked as answer]
@axtens thanks for the info. So, the Resizetizer isn't overrinding your `ApplicationIcon` property. Looking further, the winUI doesn't have support for `ApplicationIcon` property, if you try to do it on WPF head, for example, that will work. I see, from binlog, that your project uses `Uno.Resizetizer` so what you can do is to convert your `.ico` file to `.svg` and replace the `UnoIcon.Foreground` (that's defined on the `base.props` file) for the generated `svg`
